Camel-Cron connector: We need to add at least camel-quartz as dependency

Otherwise we have 

`Caused by: org.apache.camel.FailedToCreateRouteException: Failed to create route route1: Route(route1)[From[cron:cron-timer?schedule=0/5+*+*+*+*+?] -... because of Failed to resolve endpoint: cron://cron-timer?schedule=0/5+*+*+*+*+? due to: Cannot find any CamelCronService: please add a valid implementation, such as 'camel-quartz', in order to use the 'camel-cron' component`

--- a/connectors/camel-cron-kafka-connector/pom.xml
+++ b/connectors/camel-cron-kafka-connector/pom.xml
@@ -44,6 +44,10 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-cron</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-quartz</artifactId>
+    </dependency>
     <!--START OF GENERATED CODE-->
     <dependency>
       <groupId>org.apache.camel.kafkaconnector</groupId>

